examples/kill-random-pod.js

```diff
@@ -0,0 +1,26 @@
+import { sleep } from 'k6';
+import { Kubernetes } from 'k6/x/kubernetes';
+import { KubernetesChaosClient } from '../src/chaos.js';
+
+export default function () {
+
+  // KubernetesChaosClient requires the Kubernetes client from xk6-kubernetes.
+  const k8sClient = new Kubernetes()
+
+  // get number of existing pods
+  let pods = k8sClient.pods.list(),
+      podsLength = pods.length;
+
+
+  // randomly kill a pod of the given namespace.
+  KubernetesChaosClient.killRandomPod(k8sClient, pods[0].namespace);
```

Comment:
It's not random, though, we just use the first one. 

We can improve the examples by using a random int between 0 and the lenght-1 of the pod's list. This applies to all the other examples.

Reply:
I might add that these `kill...` functions should potentially return the full object rather than just the name. That way, if someone wanted to re-add the resource after deleting, they would have all the information necessary to recreate it. WDYT?

Reply:
I love that. Objects everywhere! Fully agree.

Reply:
@javaducky It didn't work, does `xk6-kubernetes` return the deleted object?

Reply:
Merged. let's create a new issue for this.